tests/test_train/test_hooks.py

```diff
@@ -86,6 +87,9 @@ def train_loop_epoch(self, length, max_it_len):
 
 
 def test_summary_hook():
+    if sys.platform.startswith('win'):
+        pytest.skip('this doctest does not work on Windows, '
+                    'training is not possible on Windows due to symlinks being unavailable')
```

Comment:
Can you explain why this is the case? E.g., add a comment or change the skip message to something like "Training is not possible on Windows because file symlinks are unavailable for non-admin users" (I believe that this is the reason, not 100% sure)

Reply:
You can add a new function below the `padertorch.testing` namespace and add there the comments. In this way, the skip message can stay short, and the source code has more details.